persistence/src/commonMain/db_user/com/wire/kalium/persistence/ConversationDetails.sq

```diff
@@ -108,7 +108,15 @@ CASE
             ELSE 1
         END
     ELSE 0
-END AS interactionEnabled
+END AS interactionEnabled,
+CASE WHEN EXISTS (
+        SELECT 1
+        FROM LabeledConversation lc
+        JOIN ConversationFolder cf ON lc.folder_id = cf.id
+        WHERE lc.conversation_id = Conversation.qualified_id
+          AND cf.folder_type = 'FAVORITE'
+    )
+    THEN 1 ELSE 0 END AS isFavorite
```

Comment:
suggestion: making multiple nested queries can be expensive.
I would suggest to change it to use joins, like:
```
LabeledConversation.folder_id IS NOT NULL AS isFavorite
FROM Conversation
LEFT JOIN ConversationFolder AS FavoriteFolder ON FavoriteFolder.folder_type = 'FAVORITE'
LEFT JOIN LabeledConversation ON LabeledConversation.conversation_id = Conversation.id AND LabeledConversation.folder_id = FavoriteFolder.id
```
First left join filters out and joins only favorite folder, second left join adds given conversation's folders but filters them so that it only adds the one which matches with favorite folder id, so in the end `LabeledConversation.folder_id` is equal to favorite folder id or NULL if this conversation is not in favorite folder 😄